rmgpy/yml.py

```diff
@@ -0,0 +1,175 @@
+import yaml
+import os
+from rmgpy.chemkin import loadChemkinFile
+from rmgpy.species import Species
+from rmgpy.reaction import Reaction
+from rmgpy.thermo.nasa import NASAPolynomial, NASA
+from rmgpy.thermo.wilhoit import Wilhoit
+from rmgpy.kinetics.arrhenius import Arrhenius, PDepArrhenius, MultiArrhenius, MultiPDepArrhenius
+from rmgpy.kinetics.falloff import Troe, ThirdBody, Lindemann
+from rmgpy.kinetics.chebyshev import Chebyshev
+from rmgpy.data.solvation import SolventData
+from rmgpy.kinetics.surface import StickingCoefficient
+from rmgpy.util import makeOutputSubdirectory
+
+def convertChemkin2yml(chemkinPath, spcDictPath=None, output="chem.rms"):
+    if spcDictPath:
+        spcs,rxns = loadChemkinFile(chemkinPath,dictionaryPath=spcDictPath)
+    else:
+        spcs,rxns = loadChemkinFile(chemkinPath)
+    writeyml(spcs,rxns,path=output)
+
+def writeyml(spcs, rxns, solvent=None, solventData=None, path="chem.yml"):
+    D = getMechDict(spcs,rxns,solvent=solvent,solventData=solventData)
+    yaml.dump(D,stream=file(path,'w'))
+
+def getMechDict(spcs, rxns, solvent='solvent', solventData=None):
+    D = dict()
+    D["Units"] = dict()
```

Comment:
Does ["Units"] ever get updated anywhere else in the code? Does the YAML format for RMS/Cantera assume that an empty dictionary for units defaults to something?

Reply:
It defaults to SI units for everything, unless units are specified for individual floats

Reply:
That works then